wled00/FX.cpp

```diff
@@ -5858,7 +5858,7 @@ uint16_t mode_2Dscrollingtext(void) {
   char text[33] = {'\0'};
   if (SEGMENT.name) for (size_t i=0,j=0; i<strlen(SEGMENT.name); i++) if (SEGMENT.name[i]>31 && SEGMENT.name[i]<128) text[j++] = SEGMENT.name[i];
 
-  if (!strlen(text) || !strncmp_P(text,PSTR("#DATE"),5) || !strncmp_P(text,PSTR("#TIME"),5)) { // fallback if empty segment name: display date and time
+  if (!strlen(text) || !strncmp_P(text,PSTR("#DATE"),5) || !strncmp_P(text,PSTR("#DDMM"),5) || !strncmp_P(text,PSTR("#TIME"),5) || !strncmp_P(text,PSTR("#HHMM"),5)) { // fallback if empty segment name: display date and time
```

Comment:
You may want to add the ugly American format MM/DD. 😉

Reply:
Done - I've added #MMDD as well in the MM/DD format. Thanks for the feedback. 

I guess that second commit got also automatically added to this pull request or? Sorry, contributing through git is super new to me :)